pkg/operator/ceph/config/monstore.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2019 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+
+	rookceph "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	"github.com/rook/rook/pkg/clusterd"
+	"github.com/rook/rook/pkg/daemon/ceph/client"
+)
+
+// MonStore provides methods for setting Ceph configurations in the centralized mon
+// configuration database.
+type MonStore struct {
+	context   *clusterd.Context
+	namespace string
+}
+
+// GetMonStore returns a new MonStore for the cluster.
+func GetMonStore(context *clusterd.Context, namespace string) *MonStore {
+	return &MonStore{
+		context:   context,
+		namespace: namespace,
+	}
+}
+
+// Set sets a config in the centralized mon configuration database.
+// https://docs.ceph.com/docs/master/rados/configuration/ceph-conf/#monitor-configuration-database
+func (m *MonStore) Set(who, option, value string) error {
+	args := []string{"config", "set", who, normalizeKey(option), value}
+	cephCmd := client.NewCephCommand(m.context, m.namespace, args)
+	out, err := cephCmd.Run()
+	if err != nil {
+		return fmt.Errorf("failed to set Ceph config in the centralized mon configuration database. output: %s. %+v", string(out), err)
```

Comment:
This should be the right place to explain that the user might need to rely on the ConfigMap`rook-config-override`, right?